nginx-php-moodle/bin/configure-moodle.sh

```diff
@@ -79,6 +79,16 @@ MDL_CONFIG
 
     color_echo green "Skipping Moodle registration"
     /usr/bin/php "${moodle_cfg_script}" --name=registrationpending --set=0
+
+    color_echo green "Configuring Logging"
+    /usr/bin/php "${moodle_cfg_script}" --name=debug --set=15
+    /usr/bin/php "${moodle_cfg_script}" --name=debugdisplay --set=0
+
+    color_echo green "Configuring Email settings"
+    /usr/bin/php "${moodle_cfg_script}" --name=smtphosts --set=postfix
+
+    color_echo green "Configuring Authentication"
+    /usr/bin/php "${moodle_cfg_script}" --name=auth --set=onelogin_saml
```

Comment:
should onelogin be the default auth provider?

Reply:
It makes more sense to me to leave it as default and configure it afterwards if you want to use onelogin or another auth provider. As far as I know not all our Moodle servers use onelogin for authentication right now.

Reply:
So leave which? My goal was to disable email based self-registration (don't ever foresee using this) but I couldn't..

Reply:
Leave it as in don't include this setting. If you enable onelogin by default then you still have to go to the UI to configure it right? So either way you have to change settings in the UI. For general use I'd imagine people using this container would want email based auth setup at first anyway.

Reply:
Hmm OK. let me see what I can do.. The `admin` account is created during installation automatically and with that you can make manual accounts.

Reply:
The other thing that worries me is if we forget to turn OFF self-registration then people can sign up and we wouldn't know..

Reply:
Hm.. so leaving that field empty does what I need without enabling OneLogin by default - sound good? Not sure why it didn't work before - I might have hit an unrelated error and thought it was the `auth` field

![image](https://user-images.githubusercontent.com/7264838/48514391-362e2900-e82c-11e8-9b95-fb1117855614.png)

Reply:
This way we don't get weird OneLogin errors (because its not configured) and don't accidentally allow users to register.

Reply:
Nice that sounds good to me

Reply:
Latest changes pushed

Reply:
Sweet, might be good to add a comment on why the value is left blank.